pkgs/racket-doc/scribblings/reference/promise.scrbl

```diff
@@ -174,3 +174,46 @@ TODO: Say something on:
   demand.
 ;}
 }
+
+@(define promise-eval
+   (let ([eval (make-base-eval)])
+     (eval '(require racket/promise))
+     eval))
+
+@defform[
+  (for/concurrent maybe-group (for-clause ...) body-or-break ... body)
+  #:grammar
+  [(maybe-group (code:line)
+                (code:line #:group thread-group-expr))]
+  #:contracts ([thread-group-expr thread-group?])]{
+
+  Iterates like @racket[for/list], but the bodies (following any
```

Comment:
I guess I'd expect the name `for/concurrent` to be like `for`, instead of like `for/list`.

Reply:
Yeah, that's part of why I don't like the name either. I think the fact that it returns a list is important, though (otherwise you'd just use a `for` with `thread`s in the body). So, it should definitely be like `for/list`, but with a better name.

Reply:
`for/conscurrent`?

Ok, I don't have a good name, either. I assume that no one is suggesting something like `for/list/concurrent` because it's too long.

Reply:
> I assume that no one is suggesting something like for/list/concurrent because it's too long.

I was thinking it would be, but now I've updated some code that uses `for/concurrent` and it's not too bad. I'll update the PR later today to go with that name.